calculus/functions.md

```diff
@@ -1 +1,73 @@
 # Functions
+### Evaluating and Manipulating Trigonometric Functions
+![fishy](./images/MAtappunitcircle.png)
+
+\begin{align}
+\sin(\theta)& = \frac{y}{r}& 
+\cos(\theta)& = \frac{x}{r}& 
+\tan(\theta)& = \frac{y}{x}\\
+\csc(\theta)& = \frac{r}{y}&
+\sec(\theta)& = \frac{r}{x}&
+\cot(\theta)& = \frac{x}{y}\\
+\end{align}
+
+Angles can be measured in degrees of radians
+\begin{align}
+\theta(rad.)& = \theta(deg.)(\frac{\pi}{180^{\circ}})
+\end{align}
+
+Special Angles
+\begin{align}
+\theta& = 30^{\circ} \frac{\pi}{6}& \theta& = 60^{\circ} \frac{\pi}{3}
+\end{align}
+
+![fishy](./images/306090triangle.png)
+
+At $ \theta = 30^{\circ} $:
+\begin{align}
+\sin(\theta)& = \frac{1}{2}& 
+\cos(\theta)& = \frac{\sqrt{3}}{2}
+\end{align}
+
+At $ \theta = 60^{\circ} $:
+\begin{align}
+\sin(\theta)& = \frac{\sqrt{3}}{2}&
+\cos(\theta)& = \frac{1}{2}& 
+\end{align}
+
+![fishy](./images/454590triangle.png)
+
+At $ \theta = 45^{\circ} $
+\begin{align}
+\sin(\theta)& = \frac{1}{\sqrt{2}}&
+\cos(\theta)& = \frac{1}{\sqrt{2}}
+\end{align}
+
+What about $ \theta' \ge 90^{\circ}(\frac{\pi}{2}) $?
+Take the angle of $\theta$ relative to the x-axis, then apply the sign based on which quadrant the angle falls in.
+
+Example: $\theta = 185^{\circ}$
```

Comment:
Examples are best put in an `{example}` directive, like so:

````
```{example} <Put title of example here>
<put example problem and content here>
```
````
There are other files in this repository that use the example directive that I would recommend looking to get an idea of how to use it.